AlmesCore/.htaccess

```diff
@@ -0,0 +1,19 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Redirect to the schema page
+RewriteRule ^$ https://github.com/AlmesCore/Almes-Core/blob/main/core2024-07-08.md [R=302,L]
+RewriteRule ^(.*)$ https://github.com/AlmesCore/Almes-Core/blob/main/core2024-07-08.md/$1 [R=302,L]
```

Comment:
Rules are checked in order.  This catch-all will handle everything before the rules below are checked.  Easy fix it to put in at the bottom.

Reply:
@davidlehn thanks for the suggestion. I just updated the code. Please let me know if there's anything else to fix.